packages/trader/src/AppV2/Components/MarketCategory/market-category.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useRef, useEffect } from 'react';
+import { MarketGroup } from 'AppV2/Utils/symbol-categories-utils';
+import { Tab, Text, CaptionText } from '@deriv-com/quill-ui';
+import MarketCategoryItem from '../MarketCategoryItem';
+import { ActiveSymbols } from '@deriv/api-types';
+import FavoriteSymbols from '../FavoriteSymbols';
+
+type TMarketCategory = {
+    category: MarketGroup;
+    selectedSymbol: string;
+    setSelectedSymbol: (input: string) => void;
+    setIsOpen: (input: boolean) => void;
+    isOpen: boolean;
+};
+
+const MarketCategory = ({ category, selectedSymbol, setSelectedSymbol, setIsOpen, isOpen }: TMarketCategory) => {
+    const itemRefs = useRef<{ [key: string]: HTMLDivElement | null }>({});
+    const timerRef = useRef<NodeJS.Timeout>();
```

Comment:
```suggestion
    const timerRef = useRef<ReturnType<typeof setTimeout>>();
```
This one's safer for us because I remember I faced an error `Type 'number' is not assignable to type 'Timeout'.` one day because our final runtime target is a browser where the timer is a number, not NodeJS where the timer is a NodeJS.Timeout. I guess it was a console error. When we turn the TS compiler fully on deriv-app, we'll get a compile error.
Basically, if we used server-side rendering, `NodeJS.Timeout` type would be safe to use.